### ISSUE TITLE
Download url not working anymore

### ISSUE THREAD

**Issue by mircobabini** [doekenorg/gf-entries-in-excel#41]
In fact it partially works; but the downloaded file is empty because of the $search_criteria in GFExcelOutput.php.

Solved this way:
```
if( ! $search_criteria['start_date'] ){
    unset( $search_criteria['start_date'] );
}
if( ! $search_criteria['end_date'] ){
    unset( $search_criteria['end_date'] );
}
```

**Comment by mircobabini** [comment]
Could be my Gravity Forms outdated version? Tested on 2.2.5

**Comment by doekenorg** [comment]
Hi @mircobabini , 

Thanks for letting me know. I've not had any complaints about this, so I'm glad you pointed this out. 
I added a fix to my next release. 

You can change `GFExcel.php` line 167 to 
```php
return array_filter($search_criteria);
```

**Comment by doekenorg** [comment]
Fixed in 1.6.1
